src/unit-test/test.py

```diff
@@ -90,135 +100,188 @@ def perform_test(
+                        if value["description"] == file_url_description:
+                            # URL
+                            is_file_in_inputs = True
+                            if "audio" in key or "audio" in value["title"].lower():
+                                # URL audio
+                                urls_files.append(
+                                    {
+                                        value["title"]: (
+                                            "audio_url",
+                                            "https://anshe.org/audio/3Weeks-080715.mp3",
+                                        )
+                                    }
+                                )
+                            elif "video" in key or "video" in value["title"].lower():
+                                # URL Video
+                                urls_files.append(
+                                    {
+                                        value["title"]: (
+                                            "video_url",
+                                            "https://upload.wikimedia.org/wikipedia/commons/8/80/Expedition_50-51_Crew_Docks_to_the_Space_Station.webm",
+                                        )
+                                    }
+                                )
+                            else:
+                                # URL Image
+                                urls_files.append(
+                                    {
+                                        value["title"]: (
+                                            "image_url",
+                                            "https://upload.wikimedia.org/wikipedia/commons/thumb/b/b6/Image_created_with_a_mobile_phone.png/1200px-Image_created_with_a_mobile_phone.png",
+                                        )
+                                    }
+                                )
+                    else:
+                        # Text
+                        texts.append({value["title"]: ("text", value["default"])})
+                else:
+                    if "audio" in key or "audio" in value["title"].lower():
+                        # Audio
+                        audios.append({value["title"]: ("audio", audio_files_to_test)})
+                    elif "video" in key or "video" in value["title"].lower():
+                        # Video
+                        videos.append({value["title"]: ("video", video_files_to_test)})
+                    else:
+                        # Image
+                        images.append({value["title"]: ("image", image_files_to_test)})
+
+            # Create all requests to send for good testing of the model
             data = {}
-            if "title" in request_body_info:
-                data[request_body_info["title"]] = request_body_info["default"]
+            for text in texts:
+                for key, value in text.items():
+                    data = merge_dicts(data, {key: value[1]})
```

Comment:
Same syntax as above possible ? 
```suggestion
            dict_to_merge = {key: value[1] for text in texts for key, value in text.items()}
            data = merge_dicts(data, dict_to_merge)
```